patch_recurrence/tools.py

```diff
@@ -107,3 +107,29 @@ def show_buckety_img(imgs, constants):
 
     # show the images
     plt.show()
+
+
+def histogram(pairs,i):
```

Comment:
**nit** - space after `,` here and elsewhere. Are you using `flake8`? Perhaps I should install a basic travis setup for this project